cohorts/load.py

```diff
@@ -213,6 +213,14 @@ class Cohort(Collection):
     responder_pfs_equals_os : bool
         Ensure that the PFS values for responders (not progressed) are equal to
         OS values.
+    check_provenance : bool
+        Verify that the cached provenance is equal to the current environment.
+    print_provenance : bool
+        Print a cached provenance summary.
```

Comment:
Technically, this is a summary of the cache provenance .. not a cached summary of the provenance files. 
Maybe say "Print a summary of cache file provenance"